datadir/nginx-preauth/nginx.conf

```diff
@@ -9,8 +9,10 @@ server {
         proxy_set_header  preauth-firstname   "test";
         proxy_set_header  preauth-lastname    "admin";
         proxy_set_header  preauth-org         "georchestra";
+        proxy_set_header  preauth-idp	      "INRAE";
+        proxy_set_header  preauth-userid      "f77ded4f-734e-4cbf-8003-627a49c0c499";
```

Comment:
What is the purpose of adding these ones ? If so, it would be relevant to also update the documentation here:
https://github.com/georchestra/georchestra-gateway/blob/main/docs/pre-authentication.adoc

Also I can't see these consumed anywhere in the gateway codebase, so they should be stripped here:
https://github.com/georchestra/georchestra-gateway/blob/main/gateway/src/main/java/org/georchestra/gateway/security/preauth/PreauthAuthenticationManager.java#L111-L119

Reply:
@pmauduit  I need IDP name and IDP user ID to be shown later.